dev-haskell/hmatrix/hmatrix-0.20.0.0.ebuild

```diff
@@ -1,33 +0,0 @@
-# Copyright 1999-2020 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=7
-
-# ebuild generated by hackport 0.6.3
-
-CABAL_FEATURES="lib profile haddock hoogle hscolour"
-inherit haskell-cabal
-
-DESCRIPTION="Numeric Linear Algebra"
-HOMEPAGE="https://github.com/albertoruiz/hmatrix"
-SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
-
-LICENSE="BSD"
-SLOT="0/${PV}"
-KEYWORDS="~amd64 ~x86"
-IUSE=""
-
-RDEPEND="dev-haskell/random:=[profile?]
-	dev-haskell/semigroups:=[profile?]
-	dev-haskell/split:=[profile?]
-	dev-haskell/storable-complex:=[profile?]
-	>=dev-haskell/vector-0.11:=[profile?]
-	>=dev-lang/ghc-8.0.1:=
-	virtual/blas
-	virtual/lapack
-"
-DEPEND="${RDEPEND}
-	>=dev-haskell/cabal-1.24.0.0
-"
-
-PATCHES=( "${FILESDIR}"/${PN}-0.20.0.0-gentoo-blas.patch )
```

Comment:
It looks like the previous author of the ebuild used this patch to remove `openblas` support entirely. You can try adding this line (and the associated patch) back in. You will also need to remove the `openblas` USE flag and the associated entry in `metadata.xml`.

If the patch fails to apply when you try building it, you can create a new patch for this version. (I can share my workflow in this regard if we get there.)